### DISCUSSION TITLE
Resize only certain holes in a model possible?

### DISCUSSION THREAD

**Discussion by pimmen85** [supermerill/SuperSlicer#3696] [General]
Would it be possible to only resize the round holes in the model attached??
![Namnlös](https://user-images.githubusercontent.com/83097086/223285549-3f647814-2dbe-4332-abec-6d21fab7744a.jpg)

**Comment by Bumblebiber** [comment]
I guess not in SuperSlicer. What you could do is to add cylinders as negative volumes and place them in each hole but you won't 100% hit the middle of each hole. 

The easiest way I know is to import the model to blender and scale up the holes there.
